src/main/app/claims/claimModelConverter.ts

```diff
@@ -58,6 +63,9 @@ export class ClaimModelConverter {
     }
 
     draftClaim.defendant['type'] = 'individual'
```

Comment:
enum?

Reply:
anirudha's PR has this completely reworked, this is temporary